igcollect/artfiles.py

```diff
@@ -1,12 +1,18 @@
-#!/usr/bin/env python
 """igcollect - Artfiles Hosting Metrics
 
-Copyright (c) 2016 InnoGames GmbH
+Copyright (c) 2019 InnoGames GmbH
 """
 
+import base64
 from argparse import ArgumentParser
 from time import time
-from urllib import urlopen
+
+try:
```

Comment:
Please add a comment which of these modules are for Python2 and which are for Python3.